src/base/bittorrent/session.cpp

```diff
@@ -537,14 +487,18 @@ Session::Session(QObject *parent)
     m_resumeDataSavingManager->moveToThread(m_ioThread);
     connect(m_ioThread, &QThread::finished, m_resumeDataSavingManager, &QObject::deleteLater);
     m_ioThread->start();
-    m_resumeDataTimer->start();
+
+    m_startupThread = new QThread(this);
```

Comment:
You can use existing m_ioThread for this purpose. Why is yet another thread needed?